packages/reports/addon/components/filter-collection.js

```diff
@@ -69,5 +69,23 @@ export default Component.extend({
     });
 
     return [...dateFilters, ...dimFilters, ...metricFilters];
-  })
+  }),
+
+  /**
+   * return appropriate dimension filter builder type based on dimension value Type.
+   * @param {String} type - dimension meta data value type.
+   * @return {String} which dimension filter builder to use
+   */
+  _dimensionFilterBuilder: function(type) {
+    if (!featureFlag('dateDimensionFilter')) {
```

Comment:
Do we still need this feature flag?

Reply:
I suppose not, it only gets triggered by specific metadata.